### DISCUSSION TITLE
Docs: Add knowledge of using next's cli favoured logger

### DISCUSSION THREAD

**Discussion by larsqa** [vercel/next.js#40899] [Ideas]
### What is the improvement or update you wish to see?

I'm running some server side startup scripts and would like to log some info or errors using Next.js's cli log flavours.

I had to dig deep in the build code to find the responsible method and was able to use it like:
```js
const nextLog = require("next/dist/build/output/log")
nextLog.info("This is an info message");
```

![image](https://user-images.githubusercontent.com/107050820/192246522-821ca991-2e4d-4a08-8f28-34643063d390.png)
From above image, the second log uses Next.js' flavoured log

Would it be possible/logical to add the existence of nextLog in the documentations?

**Comment by icyJoseph** [comment]
You could also use `npm install chalk`, and invoke it with the contents of the [file you point to](https://github.com/vercel/next.js/blob/canary/packages/next/build/output/log.ts):

```ts
import chalk from 'chalk'

export const prefixes = {
  wait: chalk.cyan('wait') + '  -',
  error: chalk.red('error') + ' -',
  warn: chalk.yellow('warn') + '  -',
  ready: chalk.green('ready') + ' -',
  info: chalk.cyan('info') + '  -',
  event: chalk.magenta('event') + ' -',
  trace: chalk.magenta('trace') + ' -',
}

export function wait(...message: any[]) {
  console.log(prefixes.wait, ...message)
}

export function error(...message: any[]) {
  console.error(prefixes.error, ...message)
}

export function warn(...message: any[]) {
  console.warn(prefixes.warn, ...message)
}

export function ready(...message: any[]) {
  console.log(prefixes.ready, ...message)
}

export function info(...message: any[]) {
  console.log(prefixes.info, ...message)
}

export function event(...message: any[]) {
  console.log(prefixes.event, ...message)
}

export function trace(...message: any[]) {
  console.log(prefixes.trace, ...message)
}
```

And then if the `next` team moves the logger to another folder or something changes, you still got your logger.

**Reply by larsqa** [reply]
This makes a lot of sense - thanks!
